package.json

```
@@ -1,51 +1,52 @@
 {
   "name": "ember-legacy-controllers",
-  "version": "0.1.0",
-  "description": "Ember.ObjectController and Ember.ArrayController extracted as an standalone addon",
+  "version": "0.0.0",
```

Should probably reset this to 0.1.0

```suggestion
  "version": "0.1.0",
```